src/uct/tcp/sockcm/sockcm_def.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2017.  ALL RIGHTS RESERVED.
+ * Copyright (C) NVIDIA Corporation. 2019.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCT_SOCKCM_H
+#define UCT_SOCKCM_H
+
+#include <uct/api/uct.h>
+#include <uct/api/uct_def.h>
+#include <uct/base/uct_iface.h>
+#include <uct/base/uct_md.h>
+#include <ucs/type/class.h>
+#include <ucs/time/time.h>
+#include <ucs/async/async.h>
+#include <sys/poll.h>
+#include <ucs/sys/sock.h>
+#include <net/if.h>
+
+#define UCT_SOCKCM_TL_NAME              "sockcm"
+#define UCT_SOCKCM_UDP_PRIV_DATA_LEN    1024   /** FIXME */
```

Comment:
do we use UDP?
i think it's better to move "uct_sockcm_conn_param_t" to next PR so we can review it in the proper context

Reply:
Done